src/main/haskell/ast/src/Data/Kore/AST.hs

```diff
@@ -0,0 +1,563 @@
+{-|
+Module      : Data.Kore.AST
+Description : Data Structures for representing the Kore language AST
+Copyright   : (c) Runtime Verification, 2018
+License     : UIUC/NCSA
+Maintainer  : traian.serbanuta@runtimeverification.com
+Stability   : experimental
+Portability : POSIX
```

Comment:
So this is not going to work on Windows natively?

Reply:
We should probably change that line... it was the default in some example, then we started copying it around.  Thanks for pointing it out!